README.md

```diff
@@ -1,11 +1,11 @@
 <img width="200" align="right" src="img/uhyve.svg" />
 
-# Uhyve - A minimal hypervisor for Hermit
+# Uhyve
 
 [![crates.io](https://img.shields.io/crates/v/uhyve.svg)](https://crates.io/crates/uhyve)
 [![Zulip Badge](https://img.shields.io/badge/chat-hermit-57A37C?logo=zulip)](https://hermit.zulipchat.com/)
 
-Uhyve is a small hypervisor specialized for the [Hermit kernel](https://github.com/hermit-os/kernel).
+Uhyve is a minimal, special-purpose hypervisor for the [Hermit kernel](https://github.com/hermit-os/kernel).
```

Comment:
perhaps "minimalistic" would be better, "minimal" sounds like a promise

- https://github.com/tandasat/SimpleSvm
- https://github.com/SamuelTulach/memhv

these hypervisors use "minimal":
- https://github.com/tenclass/mvisor
- https://github.com/flouthoc/wiser

But honestly, I think I'm the first and most likely the last person to care about that detail ever.

Reply:
I think both are fine